### PR TITLE
Web Front End

### PR DESCRIPTION
**Pull request by cynexit**
The first iteration of the front end!
Gives you the ability to create jobs, check their status and results.
This pull also adds the first iteration of the "new" API with JSON support and better routing as well as a file server for static files.

### CODE REVIEW

File: src/main/web/assets/js/app/config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"api_url": "",
+	"debug": false,
+	"services": ["ASNMETA", "CUCKOO", "DNSMETA", "GOGADGET", "OBJDUMP", "PEID", "PEINFO", "VIRUSTOTAL", "YARA", "ZIPMETA"],
```
Comment (webstergd) [inline src/main/web/assets/js/app/config.json:4]: I assume this is for being able to select services. Question is should we dynamically generate this list from the Gateway or a DB entry in Storage? Additionally, do we want to eventually allow rules based on the type of file of values returned? Last is totally a future thing. But figured I would flag this here

File: src/main/web/modules/dashboard/view.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="row">
+	<div class="col-md-12">
+		<div class="panel panel-holmes">
+			<div class="panel-heading">Welcome to the Holmes Web Interface!</div>
+			<div class="panel-body">
+				<p>This interface is currently in an early alpha stage, please be carful and don't expect everything to work flawlessly all the time.</p>
+				<p>Bugs and feature requests may be reported to the <a href="https://github.com/HolmesProcessing/Holmes-Frontend">GitHub page</a>.</p>
```
Comment (webstergd) [inline src/main/web/modules/dashboard/view.html]: need up update url

**Comment by webstergd** [comment]
are you submitting jobs by writing the AMQP or submitting through gateway?

**Review by webstergd — COMMENTED** [review]
looks good. few comments

File: src/main/web/assets/js/app/config.json (hunk 1 above)
Comment (cynexit) [reply src/main/web/assets/js/app/config.json:4]: You are fully correct here. The `config.json` file was carried over from the old front end so we can add the other front end features later easily. This data can later be received from storage, but we would have to add this as an API call.

File: src/main/web/modules/dashboard/view.html (hunk 2 above)
Comment (cynexit) [reply src/main/web/modules/dashboard/view.html]: Done

**Comment by cynexit** [comment]
Jobs are currently submitted using the HTTP API directly. This should be moved to gateway later on to have UAC, but right now gateway doesn't support it. This should be an easy add though since it's also just an HTTP request, just like uploading a new file to storage.

**Pull request merged**